google-api-client/src/main/java/com/google/api/client/googleapis/services/AbstractGoogleClientRequest.java

```diff
@@ -126,7 +126,7 @@ protected AbstractGoogleClientRequest(AbstractGoogleClient abstractGoogleClient,
     // application name
     String applicationName = abstractGoogleClient.getApplicationName();
     if (applicationName != null) {
-      requestHeaders.setUserAgent(applicationName + " " + USER_AGENT_SUFFIX);
+      requestHeaders.setUserAgent(applicationName + GoogleUtils.VERSION + " " + USER_AGENT_SUFFIX);
```

Comment:
minor - do you want to leave a space or delimiter between application name and version number?